app/assets/javascripts/application.js

```diff
@@ -17,6 +17,11 @@
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require controllers/deployment/wizard-module
+//= require controllers/deployment/wizard/wizard-directive
+//= require controllers/deployment/wizard/wizard-buttons
+//= require controllers/deployment/wizard/wizard-step-directive
+//= require controllers/deployment/wizard/wizard-substep-directive
```

Comment:
Cant we put these in directives or `app/assets/javascripts/controllers/deployment/`?